src/helpers/terraform.js

```diff
@@ -287,13 +287,12 @@ class Terraform {
 
     return this.run('plan', args.concat(this._varFile(), this._var(), this._optsToArgs(options)))
       .then(data => {
-        let metadata;
+        let metadata = {};
```

Comment:
`metadata` can be `const`

Reply:
ok, please rewrite how you think it should work.